data/recorder/src/main/java/tech/pegasys/artemis/data/recorder/SSZTransitionRecorder.java

```diff
@@ -42,7 +42,7 @@ public SSZTransitionRecorder(final Path outputDirectory) {
   }
 
   @Subscribe
-  public void onGenesisEvent(final StoreGenesisDiskUpdateEvent genesisEvent) {
+  public void onDiskGenesisUpdate(final DiskGenesisUpdate genesisEvent) {
```

Comment:
Not sure `DiskGenesisUpdate` is the right name for this event.  It's probably just `GenesisEvent`

Reply:
Done.